tests/PySys/tedge_connect/tedge_connect_test_sm_services/run.py

```diff
@@ -21,18 +21,18 @@ class TedgeConnectTestSMServices(EnvironmentC8y):
     # `sudo tedge connect c8y` is run by the super.execute()
     def execute(self):
         super().execute()
-      
+
     def validate(self):
         super().validate()
         self.log.info("Validate")
-              
-        # Validate if the Software management services are getting started and enabled properly on "tedge disconnect c8y"  
+
+        # Validate if the Software management services are getting started and enabled properly on "tedge disconnect c8y"
```

Comment:
```suggestion
        # Validate if the Software management services are getting started and enabled properly on "tedge connect c8y"
```

Reply:
Thanks I also updated the comment in line 30